frame/staking-rewards/src/test/runtime.rs

```diff
@@ -189,7 +189,23 @@ parameter_types! {
 	pub const StakingRewardsPalletId : PalletId = PalletId(*b"stk_rwrd");
 }
 
-impl pallet_staking_rewards::Config for Runtime {
+#[derive(PartialEq, Eq, Copy, Clone, Encode, Decode, MaxEncodedLen, TypeInfo, core::fmt::Debug)]
+pub struct MaxStakingDurationPresets;
+impl Get<u32> for MaxStakingDurationPresets {
+	fn get() -> u32 {
+		10
+	}
+}
+
+#[derive(PartialEq, Eq, Copy, Clone, Encode, Decode, MaxEncodedLen, TypeInfo, core::fmt::Debug)]
+pub struct MaxRewardConfigsPerPool;
+impl Get<u32> for MaxRewardConfigsPerPool {
+	fn get() -> u32 {
+		10
```

Comment:
Use GetU32<10>

Reply:
Here we need this struct to implement `TypeInfo`

Reply:
frame_support::traits::ConstU32<16>;

Reply:
Could you point to where that `ConstU16` implements `TypeInfo` ?

Reply:
Fixed the problem in a different way, thanks